### PR TITLE
iOS Release Notes June 2021

### PR DESCRIPTION
**Pull request by tjprescott**

### CODE REVIEW

File: releases/2021-06/ios.md [hunk 1]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
 
 ### Swift CLI
 
 To add the Azure SDK for iOS to your application, follow the example in [Importing Dependencies](https://swift.org/package-manager/#importing-dependencies):
 
-Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
+Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the Swift Package Manager mirror repository and the version specifier you wish to use:
```
Comment (vcolin7) [inline releases/2021-06/ios.md:30]: Same comment about adding this to the template.

File: releases/2021-06/ios.md [hunk 2]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+#### 1.1.0-beta.1(https://github.com/Azure/azure-sdk-for-ios/blob/master/sdk/communication/AzureCommunicationCalling/CHANGELOG.md#110-beta1-2021-06-04)
+
+##### New features
+- Support for CallKit (**Preview mode**)
+  - Use the api `createCallAgentWithCallKitOptions` to create `CallAgent` with `CallKit` enabled and SDK will report to `CallKit` about incoming call , outgoing calls and all other call operations like `mute`, `unmute`, `hold`, `resume` as part of the API calls. 
+  - When the app is in the killed state and incoming call is received use the api `reportToCallKit`.
+
+- `CallAgent` and `CallClient` now has `dispose` API to explicitly delete the objects instead of relying on ARC.
+
+- Get CorrelationId from `CallInfo` object in `Call` to get the id required for recording feature.
```
Comment (vcolin7) [inline releases/2021-06/ios.md:91]:
```suggestion
- Get `CorrelationId` from the `CallInfo` object in a `Call` to get the id required for the recording feature.
```

File: releases/2021-06/ios.md [hunk 3]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+##### New features
+- Support for CallKit (**Preview mode**)
+  - Use the api `createCallAgentWithCallKitOptions` to create `CallAgent` with `CallKit` enabled and SDK will report to `CallKit` about incoming call , outgoing calls and all other call operations like `mute`, `unmute`, `hold`, `resume` as part of the API calls. 
+  - When the app is in the killed state and incoming call is received use the api `reportToCallKit`.
+
+- `CallAgent` and `CallClient` now has `dispose` API to explicitly delete the objects instead of relying on ARC.
+
+- Get CorrelationId from `CallInfo` object in `Call` to get the id required for recording feature. 
+
+- Support to start recording by an ACS endpoint.
```
Comment (vcolin7) [inline releases/2021-06/ios.md:93]:
```suggestion
- Support for starting a recording by an ACS endpoint.
```

File: releases/2021-06/ios.md [hunk 4]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
-### _Version_(_Link to Changelog_)
+### Azure Communication Services Calling
+
+#### 1.1.0-beta.1(https://github.com/Azure/azure-sdk-for-ios/blob/master/sdk/communication/AzureCommunicationCalling/CHANGELOG.md#110-beta1-2021-06-04)
+
+##### New features
+- Support for CallKit (**Preview mode**)
+  - Use the api `createCallAgentWithCallKitOptions` to create `CallAgent` with `CallKit` enabled and SDK will report to `CallKit` about incoming call , outgoing calls and all other call operations like `mute`, `unmute`, `hold`, `resume` as part of the API calls. 
+  - When the app is in the killed state and incoming call is received use the api `reportToCallKit`.
+
```
Comment (vcolin7) [inline releases/2021-06/ios.md:88]: nit;
```suggestion
```

File: releases/2021-06/ios.md [hunk 5]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- `CallAgent` and `CallClient` now has `dispose` API to explicitly delete the objects instead of relying on ARC.
+
```
Comment (vcolin7) [inline releases/2021-06/ios.md:90]: nit;
```suggestion
```

File: releases/2021-06/ios.md [hunk 6]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+##### Breaking Changes
+- Changed the way in which options are instantiated for the following classes: `CreateChatThreadOptions`, `DeleteChatThreadOptions`,  `ListChatThreadsOptions`, `AddChatParticipantsOptions`, `DeleteChatMessageOptions`, `GetChatMessageOptions`, `GetChatThreadPropertiesOptions`, `ListChatMessagesOptions`, `ListChatParticipantsOptions`, `ListChatReadReceiptsOptions`, `RemoveChatParticipantOptions`, `SendChatMessageOptions`, `SendChatReadReceiptOptions`, `SendTypingNotificationOptions`, `UpdateChatMessageOptions`, `UpdateChatThreadPropertiesOptions`.
+    - old:
+        `let options = Chat.CreatChatThreadOptions()`
+    - new:  
+        `let options = CreateChatThreadOptions()`
+- Moved `AzureCommunicationChatClient.ApiVersion` to `AzureCommunicationChatClientOptions.ApiVersion`.
+- Renamed `CommunicationError` to `ChatError`
+- Removed following classes:  `CreateChatThreadResult`, `CreateChatThreadRequest`, `ChatMessage`, `ChatMessageContent`, `ChatParticipant`, `ChatMessageReadReceipt`, `ChatThreadProperties`.
+- Removed Any type in TrouterEventUtil, and create a new enum TrouterEvent
```
Comment (vcolin7) [inline releases/2021-06/ios.md:113]: nit;
```suggestion
- Removed Any type in TrouterEventUtil, and create a new enum TrouterEvent.
```

File: releases/2021-06/ios.md [hunk 7]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
 
-- Major changes only!
+##### Breaking Changes
+- Changed the way in which options are instantiated for the following classes: `CreateChatThreadOptions`, `DeleteChatThreadOptions`,  `ListChatThreadsOptions`, `AddChatParticipantsOptions`, `DeleteChatMessageOptions`, `GetChatMessageOptions`, `GetChatThreadPropertiesOptions`, `ListChatMessagesOptions`, `ListChatParticipantsOptions`, `ListChatReadReceiptsOptions`, `RemoveChatParticipantOptions`, `SendChatMessageOptions`, `SendChatReadReceiptOptions`, `SendTypingNotificationOptions`, `UpdateChatMessageOptions`, `UpdateChatThreadPropertiesOptions`.
+    - old:
+        `let options = Chat.CreatChatThreadOptions()`
+    - new:  
+        `let options = CreateChatThreadOptions()`
+- Moved `AzureCommunicationChatClient.ApiVersion` to `AzureCommunicationChatClientOptions.ApiVersion`.
+- Renamed `CommunicationError` to `ChatError`
```
Comment (vcolin7) [inline releases/2021-06/ios.md:111]: nit;
```suggestion
- Renamed `CommunicationError` to `ChatError`.
```

File: releases/2021-06/ios.md [hunk 8]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
-Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
+Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the Swift Package Manager mirror repository and the version specifier you wish to use:
 
 ```swift
-// Insert dependencies here
+// swift-tools-version:5.3
+    dependencies: [
+        ...
+        .package(name: "AzureCore", url: "https://github.com/Azure/SwiftPM-AzureCore.git", from: "1.0.0-beta.12")
+    ],
```
[the remark this answers is not in the document]
Comment (tjprescott) [reply releases/2021-06/ios.md]: Fixed.

File: releases/2021-06/ios.md [hunk 9]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
 
 ```swift
-// Insert dependencies here
+    targets: [
+        ...
+        .target(
+            name: "MyTarget",
+            dependencies: ["AzureCore", ...])
+    ]
+)
```
[the remark this answers is not in the document]
Comment (tjprescott) [reply releases/2021-06/ios.md]: Fixed.

File: releases/2021-06/ios.md [hunk 10]
```diff
@@ -74,11 +77,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- `CallAgent` and `CallClient` now has `dispose` API to explicitly delete the objects instead of relying on ARC.
+
+- Get CorrelationId from `CallInfo` object in `Call` to get the id required for recording feature. 
+
+- Support to start recording by an ACS endpoint.
+
+##### Bug fixes
+- [iOS] ACSRendererView layout is off after a device rotation https://github.com/Azure/Communication/issues/127.
+- [iOS] Resizing issue for animating streams https://github.com/Azure/Communication/issues/262.
+- Creating multiple CallAgents with same token will throw error.
```
[the remark this answers is not in the document]
Comment (tjprescott) [reply releases/2021-06/ios.md]: I just copied what Calling had put in their release notes, but I rephrased these as you suggested.

File: releases/2021-06/ios.md [hunk 11]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
 
 To add the Azure SDK for iOS to your application, follow the instructions in [Adding Package Dependencies to Your App](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app):
 
-With your project open in Xcode 11 or later, select **File > Swift Packages > Add Package Dependency...** Enter the clone URL of this repository: *https://github.com/Azure/azure-sdk-for-ios.git* and click **Next**. For the version rule, specify the exact version or version range you wish to use with your application and click **Next**. Finally, place a checkmark next to each client library you wish to use with your application, ensure your application target is selected in the **Add to target** dropdown, and click **Finish**.
```
[the remark this answers is not in the document]
Comment (tjprescott) [reply releases/2021-06/ios.md:31]: Created a separate PR for this.

**Comment by tjprescott** [comment]
@vcolin7 comments addressed. Please take another look.

File: releases/2021-06/ios.md [hunk 12]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
-Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
+Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the Swift Package Manager mirror repository and the version specifier you wish to use:
 
 ```swift
-// Insert dependencies here
+// swift-tools-version:5.3
+    dependencies: [
+        ...
+        .package(name: "AzureCommunicationChat", url: "https://github.com/Azure/SwiftPM-AzureCommunicationChat.git", from: "1.0.0-beta.12")
+    ],
```
Comment (vcolin7) [inline releases/2021-06/ios.md:37]: Are we purposefully leaving Calling out?

File: releases/2021-06/ios.md [hunk 13]
```diff
@@ -28,22 +21,32 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
 
 ```swift
-// Insert dependencies here
+    targets: [
+        ...
+        .target(
+            name: "MyTarget",
+            dependencies: ["AzureCommunicationChat", ...])
+    ]
+)
```
Comment (vcolin7) [inline releases/2021-06/ios.md:49]: Are we purposefully leaving Calling out?

**Review by vcolin7 — COMMENTED** [review]
Added a couple comments. Otherwise it looks good :)

File: releases/2021-06/ios.md (hunk 12 above)
Comment (tjprescott) [reply releases/2021-06/ios.md:37]: Yes. Calling does not support SwiftPM

File: releases/2021-06/ios.md (hunk 13 above)
Comment (tjprescott) [reply releases/2021-06/ios.md:49]: Yes

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**